azavea/raster-foundry
# List exports and download from project details

## Overview

This PR adds a tabbed interface to project details page to present both  scenes and exports.
This also allows for downloading export files from the UI for S3 exports.

This PR also introduces a `redirectTo` property on routes, which, as one might guess, allows us to define a redirect path on any route. For this PR, this property is used to provide a default tab on the project details page without having to change links all around the app. Linking to `projects.detail` will be enough to ensure the right page is presented.

The tab style that this PR introduces was also applied to the token page for consistency.

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~
~- [ ] Symlinks from new migrations present or corrected for any new migrations~

### Demo

<img width="1280" alt="screen shot 2017-06-22 at 10 24 42 am" src="https://user-images.githubusercontent.com/2442245/27439139-2b6f188c-5735-11e7-895c-23c366d28681.png">

### Notes

There is currently a bug on our export endpoint that prevents us from using the paginated response's `count` property to display a message when no exports are present. The message logic exists and once the endpoint is fixed, this issue should resolve itself.

There is also a bug on the `exports/{exportid}/files` endpoint that returns the test access file along with the actual exported imagery files. You'll see that this is filtered out. I've added a TODO in this spot so that we can remove the extra logic when the bug is fixed.

## Testing Instructions

 * Export some projects
 * See that they show up in the UI
 * Try downloading them once the export is complete

Closes #1781
